examples/co2_and_tracer_analysis.py

```diff
@@ -16,7 +16,7 @@
 # (which can be created the workflow presented in the Jupyter notebook
 # examples/notebooks/curvature_correction_walkthrough.ipynb).
 curvature_correction = daria.CurvatureCorrection(
-    config_source=image_folder + "config.json", width=2.8, height=1.5
+    config=image_folder + "config.json", width=2.8, height=1.5
```

Comment:
Brave change... I agree with the change. But we should be careful with these type of changes and in general pack them in a single PR. Because sometimes this won't be acceptable to many. Such name changes have to be thought through and potentially collected in a major update. Example: In particular the PoroTwin code will now have to updated, and I do not have the possibility to test it.

Reply:
I did not realize. Should I revert that part of the change? I completely agree that this should have been done in a separate PR, just didn't think of it yesterday.